molo/commenting/admin.py

```diff
@@ -1,8 +1,180 @@
 from django.contrib import admin
 
 from molo.commenting.models import MoloComment
+from molo.core.models import ArticlePage
 from django_comments.models import CommentFlag
+from django_comments.admin import CommentsAdmin
+from django.core.urlresolvers import reverse
+from django.core.exceptions import PermissionDenied
+from django.contrib.admin.views.main import ChangeList
+from django.shortcuts import get_object_or_404
+from django.contrib.admin.utils import unquote
+from django.contrib.contenttypes.models import ContentType
 
 
-admin.site.register(MoloComment)
+class MoloCommentAdmin(CommentsAdmin):
+    list_display = (
+        'comment', 'content', '_user', 'is_removed', 'is_reported',
+        'submit_date')
+
+    def is_reported(self, obj):
+        if (obj.flags.count() > 0):
```

Comment:
@miltontony that it will. I'll put a fix in so that only suggest_removal flags are counted